documentation/Deploy-MSBuild.md

```diff
@@ -22,3 +22,7 @@ If you cannot build or cannot deploy MSBuild on the same machine on which you wi
 ## .NET (Core) SDK
 
 Deploy-MSBuild can also patch a .NET (Core) SDK installation. Pass the `-runtime Core` argument to `Deploy-MSBuild.ps1` to ensure that it selects .NET Core MSBuild.
+
+### Unix
```

Comment:
```suggestion
### Linux
```

Reply:
^ NIT

Reply:
double-nit - should/could we have a `.NET SDK Tool` for powershell, so that folks could do `dotnet tool restore && dotnet pwsh scripts/Deploy-MSbuild.ps1` without requiring a global install?

Reply:
Imo, so far, I see that many build-related scripts exist in 2 versions - shell and PowerShell. It is not very convenient to develop both. So, making the usage of PowerShell more accessible could be a good thing. If I am wrong about that, doing that solely for the MSBuild repo is an overkill :)